src/views/projects/router.ts

```diff
@@ -101,6 +104,12 @@ export type LoadedSourceBrowsingView =
       totalCommitCount: number;
     };
 
+interface LoadedSourceBrowsingParams {
+  loadedBranches: Record<string, string> | undefined;
+  loadedPeers: Remote[];
+  loadedTree: Tree;
```

Comment:
We probably should rename these as a follow-up.

Reply:
Yes